tests/system_tests/sample_plugin/mokapot_ctree/__init__.py

```diff
@@ -12,7 +12,7 @@
 class PluginModel(Model):
     def __init__(self, *args, **kwargs):
         LOGGER.warning("The ctree model is not production ready")
-        clf = tree.DecisionTreeClassifier()
+        clf = svm.LinearSVC()
         super().__init__(clf, *args, **kwargs)
```

Comment:
quick question ... why was this changed?

Reply:
We changed the way to get the best feature from using the entire dataset to using each subset train separately, this change made the trained model not performing worse than the best feature anymore and the scores that are used are taken from the model predictions and not from the best feature.
The resulted scores of the DecisionTreeClassifier is a list of 0 and 1 which led to an error with the pep calculation using qvality.getQvaluesFromScores()

Reply:
Ohhh I see, thanks a lot